docs/operating-procedures.md

```diff
@@ -0,0 +1,75 @@
+# IDR administration
+
+If you used the OpenStack provisioning playbook the only accessible external ports will be `22`, `80` and `443` on `idr-proxy`.
+For all administrative OMERO operations you will first need to SSH into `idr-proxy` and/or setup a SSH tunnel for other ports and servers.
+
+Any changes that do not involve manipulating the data stored in OMERO (such as changes to the OMERO.server and OMERO.web configuration) should be done by modifying the deployment playbooks to ensure they are included in future deployments.
+
+
+## Production IDR
+
+There are two ways to access the production IDR:
+- Nginx web proxy running on `idr-proxy`.
+- Back-end OMERO.web and OMERO.server running on `idr-omero`.
+
+
+### `idr-proxy`
+The IDR appears as a read-only resource to public users, so the Nginx proxy has a custom caching configuration that ignores any headers sent by the back-end OMERO.web.
+This should only be used for public web access, otherwise private tokens or views may inadvertently be cached.
+
+This means if the back-end OMERO data is modified the cache may become stale.
+A special cache-buster port `9000` is configured on `idr-proxy`.
+Any requests to this port will force a cache refresh.
+In the case of large screens this may lead to a significant slowdown of the server, though it does also show the importance of the cache.
+
+You can force a complete refresh of the cache by removing everything under `/var/cache/nginx/`.
+
+
+### `idr-omero`
+If you SSH into this server you will have full access to OMERO.server, including OMERO.web and the command-line client.
+If you need to login to OMERO you should always connect to this server.
+
+If you need to restart OMERO.server or OMERO.web you must use `systemctl` and not `omero admin` or `omero web`.
+
+
+## Analysis IDR
+
+External users of the VAE should connect to JupyterHub via the front-end `idr-proxy`.
+For internal administration you may need to work on:
+- `idr-a-omero`: A clone of the production OMERO to ensure complex analysis queries do not affect public users of the IDR.
+- `idr-a-dockermanager`: The Docker server running the individual VAEs.
+
+
+### `idr-a-dockermanager`
+JupyterHub runs in a Docker container managed as a systemd service, and spawns additional Docker containers for analysis users.
```

Comment:
To clarify, the production analysis platform is currently setup for private review, so I think it's better to leave the docs until we have the final public configuration.